.github/workflows/code-checks.yml

```diff
@@ -76,29 +66,20 @@ jobs:
       # note: keep version in sync with .pre-commit-config.yaml
       run: npm install -g pyright@1.1.212
 
-    - name: Build Pandas
-      id: build
-      uses: ./.github/actions/build_pandas
-
     - name: Run checks on imported code
       run: ci/code_checks.sh code
-      if: ${{ steps.build.outcome == 'success' }}
```

Comment:
I think we would still want this to keep these checks if pandas cant be set up?

Reply:
I'm really confused about GHA's steps execution behaviour. My take was that those are redundant because GHA will stop executing the job upon the first failed step. Do you know where I can find documentation on that topic? I haven't found anything useful.